<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QWidget" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>795</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonShowBalance">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N채yt채 saldo</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonShowEvents">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selaa tilitapahtumia</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonShowWithdrawal">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nosta rahaa</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonShowCharity">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>450</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lahjoita hyv채ntekev채isyyteen</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>580</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
